CMakeLists.txt

```diff
@@ -354,13 +378,14 @@ file(GLOB_RECURSE srcs_plugins_inner CONFIGURE_DEPENDS
     "${CMAKE_CURRENT_SOURCE_DIR}/src/cradle/plugins/domain/testing/*.cpp")
 add_library(cradle_plugins_inner STATIC ${srcs_plugins_inner})
 add_dependencies(cradle_plugins_inner preprocessing)
```

Comment:
I would expect `cradle_plugins_inner` to be independent from `preprocessing` and `*.hpp`.

Reply:
There was an issue where `http_cache.cpp` included the wrong `http_requests`, so that caused me to believe there was in fact still a dependence. I fixed that include and removed this dependence from the CMake file.